ipc/apps.ts

```diff
@@ -34,6 +34,7 @@ interface Installed {
     repositoryUrl?: UrlString;
     html?: string;
     installed: {
+        publishTimestamp?: string;
```

Comment:
Whether you also need to add this, depends on what you want to do with the timestamp (which I do not know).

Reply:
Will revisit this. Initially I want to display timestamp of the already installed version of the app. Seems like the right place to add.

Reply:
👍